models/weapons/prifle/weapon.cfg

```diff
@@ -13,7 +13,6 @@ posOffs           1 5 10.25
 
 primary
 {
-  flashDlightColor  1.0 0.0 0.0
```

Comment:
Since the weapon actually fires a muzzle flash sprite, we may actually add a dynamic light that works, instead. For consistency.

The lcannon doesn't have any flash sprite displayed so it makes sense to only have the projectile having a dynamic light with the lcannon.

Reply:
Created https://github.com/Unvanquished/Unvanquished/issues/3104 to track this. I'm suffering from a lot of scope creep already with all the broken stuff I discover while reviewing all particle systems...